Makefile

```diff
@@ -14,6 +10,18 @@ check-go:
 	@./scripts/check-go.sh
 	@echo
 
+check-reqs:
+	@./scripts/check-reqs.sh
+	@echo
+
+glusterfs-csi-driver:
+	@echo Building glusterfs csi driver
+	go build -o build/glusterfs-csi-driver  cmd/glusterfs/main.go
+
```

Comment:
@humblec can we add an `@echo` after `go build`

Reply:
sure